src/Compiler/test/Microsoft.NET.Sdk.Razor.SourceGenerators.Tests/RazorSourceGeneratorTestsBase.cs

```diff
@@ -68,6 +69,7 @@ protected static async ValueTask<(GeneratorDriver, ImmutableArray<AdditionalText
         optionsProvider.TestGlobalOptions["build_property.RootNamespace"] = "MyApp";
         optionsProvider.TestGlobalOptions["build_property.RazorLangVersion"] = "Latest";
         optionsProvider.TestGlobalOptions["build_property.GenerateRazorMetadataSourceChecksumAttributes"] = "false";
+        optionsProvider.TestGlobalOptions["build_property.TestSuppressUniqueIds"] = "test";
```

Comment:
I don't love that we're routing this through the analyzer options. It feels a little weird to effectively publicly expose a test only option (even if its undocumented). 

Can we just make an internal property on the generator (or ctor param or factory method) that gets you a generator with that property set?